src/main/scala/net/fehmicansaglam/akkahttpclient/SimpleHttpRequestBuilder.scala

```diff
@@ -119,6 +122,12 @@ object SimpleHttpRequestBuilder {
   def post(uri: String): SimpleHttpRequestBuilder =
     SimpleHttpRequestBuilder(HttpRequest(method = HttpMethods.POST, uri = uri))
 
+  def post(uri: String, fields: Map[String, String]): SimpleHttpRequestBuilder =
+    SimpleHttpRequestBuilder(HttpRequest(method = HttpMethods.POST, uri = uri, entity = FormData(fields).toEntity))
+
+  def post(uri: String, fields: (String, String)*): SimpleHttpRequestBuilder =
```

Comment:
Could you move these functions to `SimpleHttpRequestBuilder` class and rename them as `bodyAsForm` or smth similar? That way we will have consistency. You may look at bodyXXX functions.